src/main/java/com/jalasoft/sdfc/pages/AppLaunchPage.java

```diff
@@ -0,0 +1,15 @@
+package com.jalasoft.sdfc.pages;
+
+import org.openqa.selenium.By;
+
+public class AppLaunchPage extends AbstractPage {
+
+    private static final String ITEM_LINK = "//a[@title='%s']";
+
+
+    public void clickOnItem(String item) {
+
+        action.click(By.xpath(String.format(ITEM_LINK, item)));
+
```

Comment:
remove the blank lines that are around the action.click ()

Reply:
removed, please review last commit.